dev/popover.html

```diff
@@ -71,19 +66,18 @@
           return;
         }
 
-        const layout = document.createElement('vaadin-horizontal-layout');
-        layout.setAttribute('theme', 'spacing');
-        layout.style.alignItems = 'end';
-
-        const field = document.createElement('vaadin-text-field');
-        field.label = 'Discount code';
+        const input = document.createElement('input');
 
-        const button = document.createElement('vaadin-button');
-        button.textContent = 'Apply';
+        const nested = document.createElement('vaadin-popover');
```

Comment:
Added a nested popover to make it easier to test different combination of triggers.